macroslib/src/java_jni/rust_code.rs

```diff
@@ -50,7 +50,7 @@ pub(in crate::java_jni) fn generate_rust_code(
 
     let mut gen_code = Vec::<TokenStream>::new();
     let (this_type_for_method, code_box_this) =
-        if let (Some(this_type), Some(constructor_ret_type)) = (
+        if let (Some(this_type), Some(_constructor_ret_type)) = (
```

Comment:
If there is no need for _constructor_ret_type why bother to take it from expression?